server/server.js

```diff
@@ -37,3 +37,5 @@ app.use('/api', apiRoutes);
 app.listen(config.port, config.host, () => {
   console.info(`Express is running on: ${config.serverUrl()}`);
 });
+
+app.use('/admin', require('./routes/routes')(app));
```

Comment:
Since we are using webpack dev server only /api calls are forwarded to server. so /admin has to be handled under /api.
@in42 can confirm, though i am mostly sure

Reply:
Also @devanshu0987  just to clarify require('./routes/routes') returns a function which  i see isn't using a parameter so why the app argument ??

Reply:
I left that part because the way I set up this prefix isn't similar to what @in42 did. I have set up it in a more modular way.

Reply:
Sorry . it shouldn't pass in the app parameter. I forgot to remove that.

Reply:
It might be a naive question like I mentioned but I don't see the point of not providing an explanation and instead reacting with an emoji

Reply:
No I mean unless the request comes as /api/admin it wont be send to the server most probably. I agree admin level modularity is  better bt handle in under api routes.

Reply:
I have tested this code with URLs like GET localhost:3000/admin/posts and it is sent to the server and it works. I feel you want it to be like GET localhost:3000/api/admin/posts?

Reply:
What I had in mind that once Vikram refactors his code to the structure I have put up, we can bring all routes under the same /api prefix. I didn't touch his code because I could have made mistakes.

Reply:
Both the /auth and /api calls will be forwarded to the node server

Reply:
GET localhost:3000/admin/posts you made this call while running the webpack dev server?

Reply:
refactor admin under api cause both  /api /auth are passed to node server. otherwise i wd nt be able to make ajax requests